init/hive/core-site.xml

```diff
@@ -0,0 +1,51 @@
+<configuration>
+  <property>
+    <name>fs.defaultFS</name>
+    <value>hdfs://__REPLACE__HOST_NAME:9000</value>
+  </property>
+
+  <property>
+    <name>name</name>
+    <value>Development Cluster</value>
+  </property>
+
+  <property>
+    <name>hadoop.http.staticuser.user</name>
+    <value>hadoopuser</value>
+  </property>
```

Comment:
I think maybe we didn't need this configure ?
I'm not sure, Please check it.

Reply:
This should be added by @yuqi1129 , Could you give some suggestion?